src/app/core/store/customer/user/user-newsletter.effects.ts

```diff
@@ -0,0 +1,74 @@
+import { Injectable } from '@angular/core';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { Store, select } from '@ngrx/store';
+import { concatMap, filter, map, switchMap, withLatestFrom } from 'rxjs/operators';
+
+import { NewsletterService } from 'ish-core/services/newsletter/newsletter.service';
+import { getServerConfigParameter } from 'ish-core/store/core/server-config';
+import { mapErrorToAction, mapToPayload, whenTruthy } from 'ish-core/utils/operators';
+
+import { userNewsletterActions, userNewsletterApiActions } from './user.actions';
+import { getLoggedInUser, getNewsletterSubscriptionStatus } from './user.selectors';
+
+@Injectable()
+export class UserNewsletterEffects {
+  constructor(private actions$: Actions, private store: Store, private newsletterService: NewsletterService) {}
+
+  /**
+   * The newsletter-subscription-status is only loaded when the feature-toggle "newsletterSubscriptionEnabled"
+   * is enabled.
+   */
+  loadUserNewsletterSubscription$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(userNewsletterActions.loadUserNewsletterSubscription),
+      withLatestFrom(
+        this.store.pipe(select(getServerConfigParameter<boolean>('marketing.newsletterSubscriptionEnabled')))
+      ),
```

Comment:
move the configuration parameter check to a facade function (see above) otherwise the loading will never be set to false again